src/algebra/group_ring_action.lean

```diff
@@ -65,6 +65,19 @@ def mul_semiring_action.to_ring_equiv [mul_semiring_action G R] (x : G) : R ≃+
 { .. distrib_mul_action.to_add_equiv R x,
   .. mul_semiring_action.to_ring_hom G R x }
 
+/-- A multiplicative action of `N` on `B` and a ring homomorphism `K →+* N` induce
+a multiplicative action of `K` on `B`. -/
+
+def mul_semiring_action.comp_hom (g : K →+* N) [mul_semiring_action N B] :
```

Comment:
I'm pretty sure you don't need `g` to be a morphism of rings here.

Reply:
OK, but I thought you suggested in (https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/ring_mul_action/near/302330592) that mul_semiring_action.comp_hom is to match mul_action.comp_hom and there is a group homomorphism in there.